lib/event.dart

```diff
@@ -1,21 +1,57 @@
-class EventMetadata {
+class ConnectorSDKCallbackMetadata {
   String connectorId;
   String? connectionId;
   String? moveId;
```

Comment:
I noticed `moveId` isn't present in [ConnectorSDKCallbackMetadata in browser.ts](https://github.com/quiltt/quiltt-public/blob/abc04d8de57b448dddbe4b0656aa5f57bd0b7d7c/ECMAScript/core/src/api/browser.ts#L68), why is that? @sirwolfgang

Reply:
Because nothing returns `moveId`. Idk what that's from.

Reply:
ok, probably a miscommunication on `moveId`, removing.